cmd/agent/dist/conf.d/ntp.d/conf.yaml.default

```diff
@@ -12,11 +12,20 @@ instances:
     #
     # offset_threshold: 60
 
-    ## @param host - string - optional - default: ntp host of cloud provider if detected, else <X>.datadog.pool.ntp.org
-    ## NTP host to connect to, default is the private NTP server of the Cloud Provider if one is detected, otherwise
+    ## @param hosts - list of strings - optional - default: ntp host of cloud provider if detected, else <X>.datadog.pool.ntp.org
+    ## List of NTP hosts to connect to, default is the private NTP server of the Cloud Provider if one is detected, otherwise
     ## `<X>.datadog.pool.ntp.org`, where <X> is a number between 0 and 3.
     #
-    # host: <X>.datadog.pool.ntp.org
+    # hosts:
+    #   - 0.datadog.pool.ntp.org
+    #   - 1.datadog.pool.ntp.org
+    #   - 2.datadog.pool.ntp.org
+    #   - 3.datadog.pool.ntp.org
+
+    ## @param host - string - optional
+    ## Single NTP host to connect to.
```

Comment:
We should write down what the default is in the description. If it's not `time.windows.com`, we should change it in this commented out template value too